src/contracts/pods/EigenPod.sol

```diff
@@ -65,6 +66,9 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     /// @notice This is the genesis time of the beacon state, to help us calculate conversions between slot and timestamp
     uint64 public immutable GENESIS_TIME;
 
+    bytes32 public immutable RANGE_SPLITTER_FUNCTION_ID = sha256(bytes("RANGE_SPLITTER_FUNCTION_ID"));
```

Comment:
Similar to the Succinct Gateway address, I think it's best if these  can be changed by `onlyOwner`. We do have a way for people to update the function_id through our gateway, but it might be cleaner for you guys if it's just all under the same system in the same file.